aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -1056,10 +1056,9 @@ Tensor compute_T12(const Tensor& A) {
   // and move them to device `device_of(A)`
   auto bs = at::from_blob(
     reinterpret_cast<void*>(&b),
-    {num_prods, num_prods},
-    {num_prods, 1},
-    A.dtype()
-  );
+    num_prods * num_prods,
+    c10::toValueType(A.scalar_type())
+  ).reshape({num_prods, num_prods});
```

Comment:
nit: does it make a difference performance wise to use `reshape` vs `view`? Reshape might be doing extra check to know if he should copy or not?

Reply:
It is a 25 by 25 array tops, should not be that huge of a difference. Does it make sense to switch to view?

Reply:
It doesn't go through the whole array for sure. But it will check size/strides I guess.
What is the reason you split this into two ops in the first place?

Reply:
Convinced, will return to my previous code then.